attachment_s3/controllers/main.py

```diff
@@ -0,0 +1,63 @@
+import logging
+import boto3
+
+from odoo.addons.web.controllers.main import Database
+from odoo import http
+from odoo import exceptions
+from odoo.http import request
+
+_logger = logging.getLogger(__name__)
+
+
+class Database(Database):
+    @http.route()
+    def drop(self, master_pwd, name):
+        response = super(Database, self).drop(master_pwd, name)
+        try:
+            bucket = request.env["ir.attachment"]._get_s3_bucket()
+            sql = ("""
+                UPDATE ir_attachment AS t SET store_fname = s.store_fname FROM (
+                    SELECT
+                        id,
+                        REPLACE(store_fname, '/*production-master*/', '%s')
+                    AS store_fname FROM ir_attachment WHERE db_datas is NULL)
+                AS s(id,store_fname) where t.id = s.id;
+            """ (bucket.name,))
+            request.env.cr.execute(sql)
```

Comment:
@hiren-serpentcs The database is already dropped.

Reply:
@hiren-serpentcs We don't need this. The database is being dropped, why would we update it?